<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gearbox</class>
 <widget class="QWidget" name="gearbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>652</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>652</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>1080</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background:transparent;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>30</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>240</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>240</width>
          <height>240</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/显示_参数.png);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_6" native="true">
           <property name="styleSheet">
            <string notr="true">background:none;</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_9" native="true">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="gearbox_rate">
                 <property name="styleSheet">
                  <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;
background:transparent;</string>
                 </property>
                 <property name="text">
                  <string>000</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignBottom|Qt::AlignHCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="maximumSize">
                  <size>
                   <width>60</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 20px;
color: #FFFFFF;
background:transparent;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 40px;
color: #FFFFFF;
background:transparent;</string>
           </property>
           <property name="text">
            <string>变速比</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>240</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>240</width>
          <height>240</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/显示_参数.png);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_7" native="true">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background:none;</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="gearbox_in_rev">
              <property name="styleSheet">
               <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;
background:transparent;</string>
              </property>
              <property name="text">
               <string>000</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignHCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 20px;
color: #FFFFFF;
background:transparent;</string>
              </property>
              <property name="text">
               <string>   rpm</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 40px;
color: #FFFFFF;
background:transparent;</string>
           </property>
           <property name="text">
            <string>输入轴转速</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>240</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>240</width>
          <height>240</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/显示_参数.png);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_8" native="true">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background:none;</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="gearbox_out_rev">
              <property name="styleSheet">
               <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;
background:transparent;</string>
              </property>
              <property name="text">
               <string>000</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignHCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 20px;
color: #FFFFFF;
background:transparent;</string>
              </property>
              <property name="text">
               <string>   rpm</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="styleSheet">
            <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 40px;
color: #FFFFFF;
background:transparent;</string>
           </property>
           <property name="text">
            <string>输出轴转速</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_10" native="true">
     <property name="styleSheet">
      <string notr="true">background:transparent;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>30</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_11" native="true">
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>240</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>240</width>
          <height>240</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/显示_参数.png);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QWidget" name="widget_13" native="true">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background:none;</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QLabel" name="oil_temper">
              <property name="styleSheet">
               <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;
background:transparent;</string>
              </property>
              <property name="text">
               <string>000</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignHCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image:url(:/temp.png)</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="styleSheet">
            <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 40px;
color: #FFFFFF;
background:transparent;</string>
           </property>
           <property name="text">
            <string>油温</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_12" native="true">
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>240</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>240</width>
          <height>240</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/显示_参数.png);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QWidget" name="widget_14" native="true">
           <property name="styleSheet">
            <string notr="true">background:none;</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="QLabel" name="oil_pressure">
              <property name="styleSheet">
               <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;
background:transparent;</string>
              </property>
              <property name="text">
               <string>000</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignHCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_13">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>20</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 20px;
color: #FFFFFF;
background:transparent;</string>
              </property>
              <property name="text">
               <string>    kPa</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="styleSheet">
            <string notr="true">font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 40px;
color: #FFFFFF;
background:transparent;</string>
           </property>
           <property name="text">
            <string>油压</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="maximumSize">
      <size>
       <width>1080</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="minimumSize">
         <size>
          <width>1000</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/背景_标题2.png);
font: 75 9pt &quot;Microsoft YaHei UI&quot;;
font-size: 60px;
color: #FFFFFF;</string>
        </property>
        <property name="text">
         <string>变速箱</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_back">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/按键_上一页2.png);
border:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
